lib/lotus/utils/escape.rb

```diff
@@ -0,0 +1,487 @@
+        8713 => 'notin',    # not an element of
+        8715 => 'ni',       # contains as member
+        8719 => 'prod',     # n-ary product
+        8721 => 'sum',      # n-ary summation
+        8722 => 'minus',    # minus sign
+        8727 => 'lowast',   # asterisk operator
+        8730 => 'radic',    # square root
+        8733 => 'prop',     # proportional to
+        8734 => 'infin',    # infinity
+        8736 => 'ang',      # angle
+        8743 => 'and',      # logical and
+        8744 => 'or',       # logical or
+        8745 => 'cap',      # intersection
+        8746 => 'cup',      # union
+        8747 => 'int',      # integral
+        8756 => 'there4',   # therefore
+        8764 => 'sim',      # tilde operator
+        8773 => 'cong',     # congruent to
+        8776 => 'asymp',    # almost equal to
+        8800 => 'ne',       # not equal to
+        8801 => 'equiv',    # identical toXCOMMAX equivalent to
+        8804 => 'le',       # less-than or equal to
+        8805 => 'ge',       # greater-than or equal to
+        8834 => 'sub',      # subset of
+        8835 => 'sup',      # superset of
+        8836 => 'nsub',     # not a subset of
+        8838 => 'sube',     # subset of or equal to
+        8839 => 'supe',     # superset of or equal to
+        8853 => 'oplus',    # circled plus
+        8855 => 'otimes',   # circled times
+        8869 => 'perp',     # up tack
+        8901 => 'sdot',     # dot operator
+        8968 => 'lceil',    # left ceiling
+        8969 => 'rceil',    # right ceiling
+        8970 => 'lfloor',   # left floor
+        8971 => 'rfloor',   # right floor
+        9001 => 'lang',     # left-pointing angle bracket
+        9002 => 'rang',     # right-pointing angle bracket
+        9674 => 'loz',      # lozenge
+        9824 => 'spades',   # black spade suit
+        9827 => 'clubs',    # black club suit
+        9829 => 'hearts',   # black heart suit
+        9830 => 'diams',    # black diamond suit
+      }.freeze
+
+      # Escape HTML contents
+      #
+      # This MUST be used only for tag contents.
+      # Please use `html_attribute` for escaping HTML attributes.
+      #
+      # @param input [String] the input
+      #
+      # @return [String] the escaped string
+      #
+      # @since x.x.x
+      #
+      # @see https://www.owasp.org/index.php/XSS_%28Cross_Site_Scripting%29_Prevention_Cheat_Sheet OWASP XSS Cheat Sheet Rule #1
+      #
+      # @example Good practice
+      #   <div><%= Lotus::Utils::Escape.html('<script>alert(1);</script>') %></div>
+      #   <div>&lt;script&gt;alert(1);&lt;&#x2F;script&gt;</div>
+      #
+      # @example Bad practice
+      #   # WRONG Use Escape.html_attribute
+      #   <a title="<%= Lotus::Utils::Escape.html('...') %>">link</a>
+      def self.html(input)
+        result = ""
+
+        input.to_s.encode(Encoding::UTF_8).chars do |chr|
+          result << HTML_CHARS.fetch(chr, chr)
+        end
+
+        result
+      end
```

Comment:
@AlfonsoUceda Because `#each_with_object` is slower: https://gist.github.com/jodosha/33c6f9201eaa4e4a5f63